dl4j-labs/src/main/resources/config.properties

```diff
@@ -3,8 +3,8 @@
 #data directory
```

Comment:
#dl4j-cv-labs properties 
change to  
#dl4j-labs properties  to avoid confusion